taxcalc/tests/test_parameters.py

```diff
@@ -164,41 +189,17 @@ def test_json_file_contents(tests_path, fname):
                    'STD', 'II_em', 'II_em_ps',
                    'AMT_em', 'AMT_em_ps', 'AMT_em_pe',
                    'ID_ps', 'ID_AllTaxes_c']
-    long_known_years = 2026 - first_year + 1  # for TCJA-reverting long_params
-    # read JSON parameter file into a dictionary
-    path = os.path.join(tests_path, '..', fname)
-    pfile = open(path, 'r')
-    allparams = json.load(pfile)
-    pfile.close()
-    assert isinstance(allparams, dict)
+    # for TCJA-reverting long_params
+    long_known_years = set(range(first_year, 2026 + 1))
     # check elements in each parameter sub-dictionary
     failures = ''
+    with open(os.path.join(tests_path, "..", fname)) as f:
+        allparams = json.loads(f.read())
     for pname in allparams:
-        # all parameter names should be strings
-        assert isinstance(pname, str)
+        if pname == "schema":
+            continue
         # check that param contains required keys
         param = allparams[pname]
-        assert isinstance(param, dict)
-        for key in required_keys:
-            assert key in param
-        if param['value_type'] == 'string':
-            for key in invalid_keys:
-                assert key not in param
-            assert isinstance(param['valid_values']['options'], list)
-        else:
-            for key in invalid_keys:
-                assert key in param
-            assert param.get('invalid_action', 'stop') in ['stop', 'warn']
-        # check for non-empty long_name and description strings
-        assert isinstance(param['long_name'], str)
-        if not param['long_name']:
-            assert '{} long_name'.format(pname) == 'empty string'
-        assert isinstance(param['description'], str)
-        if not param['description']:
-            assert '{} description'.format(pname) == 'empty string'
-        # check that indexable and indexed are boolean
-        assert isinstance(param.get('indexable', False), bool)
-        assert isinstance(param.get('indexed', False), bool)
```

Comment:
@Peter-Metz All of these checks are enforced by ParamTools when [loading the default parameters](https://github.com/PSLmodels/ParamTools/blob/0b11d235b58c3e8d2847382437201703a4e433f8/paramtools/schema_factory.py#L94).